_commands/tools/units.md

```diff
@@ -3,22 +3,37 @@
 
 units
 ---
-TODO: Add documentation for this command by submitting a pull request.
-<!-- one line explanation would go here -->
+`units` converts quantities from one unit to another unit. It can be used interactively or non-interactively.
+
 
-<!-- minimal example -->
 ~~~ bash
-*short* example of command usage and output
+$ units '10 meters' 'feet'
+# displays * 32.808399 
+           / .03048
 ~~~
 
 <!--more-->
 
 ### Useful Options / Examples
 
-#### Example command
+To view available units, look in /usr/share/units/definitions.units
+
+### Using `units` Interactively
+Brings up an interactive prompt. The line with the * gives the result. The line with the / gives the inverse of the conversion factor. In the example below, 10 meters in 32.808399 feet, and 1 foot is .03038 decameters.
 
-##### Break it down
+~~~ bash
+$ units
+$ You have: 10 meters
+$ You want: feet
+  * 32.808399
+  / .03048
+~~~
 
-#### Example command
+### Using `units` Non-interactively
+Converts the temperature from fahrenheit to celcius using one line.
+
+~~~ bash
+$ units 'tempF(32)' 'tempC'
```

Comment:
I would suggest sticking to the same example throughout the document if you're also going to be changing the syntax like this, i.e. sticking to just `$ units '10 meters' 'feet'`. Although it could also be important to mention how non-linear conversions work, like with what you have here